swagger.yaml

```diff
@@ -4252,6 +4253,36 @@ definitions:
         description: Indication of state of an NTP source.
         $ref: "#/definitions/source_state"
 
+  container_image_availability_request:
+    type: object
+    required:
+      - images
+    properties:
+      images:
+        type: array
+        description: List of image names to be checked.
+        items:
+          type: string
+          description: A fully qualified image name (FQIN).
+
+  container_image_availability_response:
+    type: object
+    required:
+      - images
+    properties:
+      images:
+        type: array
+        description: List of images that were checked.
+        items:
+          type: object
+          properties:
+            name:
+              type: string
+              description: A fully qualified image name (FQIN).
+            is_success:
```

Comment:
Will we ever want anything other than success and fail?  If there is any possibility, we should have something like
```
result:
  type: string
  enum: ['success', 'failure']
```